src/style.css

```diff
@@ -0,0 +1,103 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  font-style: normal;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.label::before {
+  position: absolute;
+  content: "";
+}
+
+.label-big::before {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.label-big:hover::before {
+  cursor: pointer;
+}
+
+.label-small::before {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.label-small:hover::before {
+  cursor: pointer;
+}
+
+.input {
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+}
+
+.input-big {
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+}
+
+.input-small {
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+}
```

Comment:
same for the styles (checklist, point 2)

Reply:
in `style.css` file i don' see the problem